dessia_common/forms.py

```diff
@@ -48,23 +48,43 @@
 from dessia_common.files import BinaryFile, StringFile
 
 
-class StandaloneSubobject(PhysicalObject):
-    _standalone_in_db = True
-    _generic_eq = True
+class EmbeddedBuiltinsSubobject(PhysicalObject):
+    """
+    An object that is not standalone and gather builtins types (float, int, bool, str & Distance).
+
+    :param distarg: A Distance with units
+    :type distarg: Distance
+    :param floatarg: A float
+    :type floatarg: float
+    :param intarg: An integer
+    :type intarg: int
+    :param boolarg: A boolean
+    :type boolarg: bool
+    :param name: Object's name
+    :type name: str
+    """
+    _standalone_in_db = False
 
-    def __init__(self, floatarg: Distance, name: str = 'Standalone Subobject'):
+    def __init__(self, distarg: Distance, floatarg: float, intarg: int,
+                 boolarg: bool, name: str = 'Standalone Subobject'):
+        self.distarg = distarg
         self.floatarg = floatarg
+        self.intarg = intarg
+        self.boolarg = boolarg
 
         PhysicalObject.__init__(self, name=name)
 
     @classmethod
-    def generate(cls, seed: int) -> 'StandaloneSubobject':
-        floatarg = Distance(1.7 * seed)
+    def generate(cls, seed: int) -> 'EmbeddedBuiltinsSubobject':
+        floatarg = 0.3
+        distarg = Distance(1.7 * floatarg * seed)
+        intarg = seed
```

Comment:
Idem, why not make a randint and get rid of seed argument ?

Reply:
The seed arg does have a lot of pros, actually. At some point, I am building a boolean over the fact that seed is even or not. So it gives us ways to check some stuff for example in serialization/deserialization, depending on a certain value, and this even on the platform.